modules/osd/bar/index.ts

```diff
@@ -13,19 +14,29 @@ export const OSDBar = (ort: OSDOrientation) => {
                 bar_mode: "continuous",
                 setup: self => {
                     self.hook(brightness, () => {
+                        self.class_names = self.class_names.filter(c => c !== "overflow");
                         self.value = brightness.screen;
                     }, "notify::screen")
                     self.hook(brightness, () => {
+                        self.class_names = self.class_names.filter(c => c !== "overflow");
                         self.value = brightness.kbd;
                     }, "notify::kbd")
                     self.hook(audio.microphone, () => {
                         self.toggleClassName("overflow", audio.microphone.volume > 1)
                         self.value = audio.microphone.volume <= 1 ? audio.microphone.volume : audio.microphone.volume - 1;
                     }, "notify::volume")
+                    self.hook(audio.microphone, () => {
+                        self.toggleClassName("overflow", audio.microphone.volume > 1 && (!options.theme.osd.muted_zero.value || !(audio.microphone.is_muted !== false)));
```

Comment:
```suggestion
                        self.toggleClassName("overflow", audio.microphone.volume > 1 && (!options.theme.osd.muted_zero.value || audio.microphone.is_muted === false));
```
We can simplify this logic so in the scenario that muted_zero IS enabled, overflow should only happen if `is_muted` is false (statement yields true). If it's null or true then that evaluates to false which won't execute the overflow. We don't have to double negate here.